src/builder/pod/volume.rs

```diff
@@ -346,6 +360,23 @@ impl SecretOperatorVolumeSourceBuilder {
     }
 }
 
+#[derive(Clone)]
+pub enum SecretFormat {
```

Comment:
Maybe some docs would be useful here.
To me it's not clear what those are.
Pkcs12 is the only real "format" here.

What would I get with Tls and kerberos?

Reply:
Yeah that's fair. They correspond to the secret-op formats (https://docs.stackable.tech/home/stable/secret-operator/secretclass.html#format, the secret-op PR adds TLS/PKCS#12 to the list as well).

Basically, `Tls` will give you a triple of PEM files as per K8s convention (`ca.crt`, `tls.crt`, `tls.key`), and `Kerberos` will give you a Kerberos keytab and associated `krb5.conf`.

Selecting an invalid combination (like requesting a Kerberos secretclass as PKCS#12) will fail and give you an error message.

Reply:
Ah! Understood. Yeah, a reference back to that page or something similar would be useful.

Reply:
Looks good to me, thank you! I'll resolve this and will let others comment on the rest